src/style.css

```diff
@@ -1 +1,63 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/174826524-89dba5d3-ea22-4609-b1cb-1026942018d7.png)
box-shadow is missing on the DEMO PAGE, have you re-deployed your DEMO PAGE?

Reply:
no, I don`t